viadot/tasks/outlook.py

```diff
@@ -55,6 +55,8 @@ def run(
             mailbox_name (str): Mailbox name.
             start_date (str, optional): A filtering start date parameter e.g. "2022-01-01". Defaults to None.
             end_date (str, optional): A filtering end date parameter e.g. "2022-01-02". Defaults to None.
+            outbox_list (List[str], optional) List of outbox folders to differenciate between
```

Comment:
Add `:` after `, optional)`. Add a dot at the end.

Reply:
done!